### PR TITLE
[2.4] add the missing rbac rules for exporter-kube-state

### PR DESCRIPTION
**Pull request by jiaqiluo**
https://github.com/rancher/rancher/issues/27748

In monitoring `v0.1.1`, the new image `rancher/coreos-kube-state-metrics:v1.9.5` for Workload `exporter-kube-state-cluster-monitoring` ask for listing/watching the following resouces:

```
  "volumeattachments" in API group "storage.k8s.io" at the cluster scope 
  "mutatingwebhookconfigurations" in API group "admissionregistration.k8s.io" at the cluster scope 
  "validatingwebhookconfigurations" in API group "admissionregistration.k8s.io" at the cluster scope 
  "networkpolicies" in API group "networking.k8s.io" at the cluster scope 
```

the fix is in the new monitoring chart `v0.1.2`

### CODE REVIEW

**Comment by StrongMonkey** [comment]
@jiaqiluo There are a lot of space changes in your change. Can you separate those into two commits, one for actual change and one for proper formatting?

**Comment by jiaqiluo** [comment]
@StrongMonkey They are in separate commits now.

**Review by StrongMonkey — DISMISSED** [review]

File: charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml [hunk 1]
```diff
@@ -86,6 +87,15 @@ rules:
...
     - storageclasses
+    - volumeattachments
+  verbs:
+  - "list"
+  - "watch"
+- apiGroups:
+  - "admissionregistration.k8s.io"
+  resources:
+    - validatingwebhookconfigurations
+    - mutatingwebhookconfigurations
```
Comment (aiyengar2) [inline charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml]:
```suggestion
  resources:
  - validatingwebhookconfigurations
  - mutatingwebhookconfigurations
```

File: charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml [hunk 2]
```diff
@@ -86,6 +87,15 @@ rules:
   - "storage.k8s.io"
   resources:
     - storageclasses
+    - volumeattachments
```
Comment (aiyengar2) [inline charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml]:
```suggestion
  resources:
  - storageclasses
  - volumeattachments
```

File: charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml (hunk 1 above)
Comment (jiaqiluo) [reply charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml]: fixed.

File: charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml (hunk 2 above)
Comment (jiaqiluo) [reply charts/rancher-monitoring/v0.1.2/charts/exporter-kube-state/templates/rbac.yaml]: fixed.

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Review by StrongMonkey — APPROVED** [review]

**Review by dramich — APPROVED** [review]

**Pull request merged**